examples/echo/main.go

```diff
@@ -107,3 +128,55 @@ func main() {
 	// Block forever
 	select {}
 }
+
+// setCodecsFromOffer finds all codecs in a session description and adds them to a MediaEngine, using dynamic
```

Comment:
What do you think of adding a `populateMediaEngineFromSDP` for lack of a better name?

Otherwise people are going to copy this around a lot, that might be ok though if you think this is an uncommon use case. We can always push it into the library later.

Also we can/should go back and update other examples using this new helper function. Will be a big improvement (I assume `gstreamer-send` will need it!)

Reply:
Will do. I'm terrible at naming things, so am always quite happy to change names of stuff. I haven't gotten to any other examples yet, but will start looking at them now.

Reply:
awesome! So I know https://github.com/pion/example-webrtc-applications/blob/master/gstreamer-send/main.go will probably blow up.

Also https://github.com/pion/webrtc/tree/master/examples/sfu-minimal probably